test/suite/wtbackup.py

```diff
@@ -92,22 +97,21 @@ def populate(self, objs, do_checkpoint=False, skiplsm=False):
     # iteration and an incremental backup for each iteration. That way we can compare the full and
     # incremental each time through.
     #
-    def setup_directories(self, max_iteration, home_incr, home_full, logpath):
-        for i in range(0, max_iteration):
-            # The log directory is a subdirectory of the home directory,
-            # creating that will make the home directory also.
+    def setup_directories(self, home_incr, home_full):
+        if not os.path.exists(self.home_tmp):
+            os.makedirs(self.home_tmp + '/' + self.logpath)
 
-            home_incr_dir = home_incr + '.' + str(i)
-            if os.path.exists(home_incr_dir):
-                os.remove(home_incr_dir)
-            os.makedirs(home_incr_dir + '/' + logpath)
+        # The log directory is a subdirectory of the home directory,
+        # creating that will make the home directory also.
+        if os.path.exists(home_full):
+            shutil.rmtree(home_full)
+        os.makedirs(home_full + '/' + self.logpath)
 
-            if i == 0:
-                continue
-            home_full_dir = home_full + '.' + str(i)
-            if os.path.exists(home_full_dir):
-                os.remove(home_full_dir)
-            os.makedirs(home_full_dir + '/' + logpath)
+        if os.path.exists(home_incr):
+            shutil.rmtree(home_incr)
+            shutil.copytree(self.home_tmp, self.home_incr)
+        else:
+            os.makedirs(home_incr + '/' + self.logpath)
```

Comment:
This is a bit tricky (and different from the original code in this function).  I see what it's doing, but a comment explaining why would be helpful.  Perhaps a reference to where `home_tmp` is created in `take_incr_backup`, and a comment there as well referencing this.

Reply:
done!